packages/@okta/vuepress-site/docs/guides/request-user-consent/require-consent/index.md

```diff
@@ -21,7 +21,12 @@ Use the following steps to display the user consent dialog box as part of an Ope
 1. On the **Authorization Servers** tab, select **default** (Custom Authorization Server) in the table. In this example, we are enabling consent for default Custom Authorization Server scopes.
 1. Select the **Scopes** tab.
 1. Click the edit icon for the **phone** scope. The Edit Scope dialog box appears.
-1. Select **Require user consent for this scope** and click **Save**.
+
+1. Select **Require user consent for this scope**. The **Block services from requesting this scope** check box is automatically selected.
+
+    The **Block services from requesting this scope** check box strictly enforces end user consent for the scope. With this check box selected, the authorization server returns an error when this scope is requested while using the [Client Credentials](/docs/guides/implement-client-creds/overview/) grant flow. This is because there is no user involved in a 2-legged OAuth Client Credentials grant flow. If you want to allow service-to-service interactions to request this scope, clear the check box. See the [Authorization Servers API](/docs/reference/api/authorization-servers/#scope-properties) for more information on consent options.
```

Comment:
```suggestion
    The **Block services from requesting this scope** check box strictly enforces end-user consent for the scope. When this check box is selected, the authorization server returns an error when this scope is requested and while it is using the [Client Credentials](/docs/guides/implement-client-creds/overview/) grant flow. The error occurs because there is no user involved in a two-legged OAuth Client Credentials grant flow. If you want to allow service-to-service interactions to request this scope, clear the check box. See the [Authorization Servers API](/docs/reference/api/authorization-servers/#scope-properties) for more information on consent options.
```